spellmonitor.lic

```diff
@@ -110,8 +110,10 @@ spell_action = proc do |server_string|
   end
 
   # The start of one or more active spells!
-  if data =~ %r{^<pushStream id="percWindow"/>}
+  if data =~ %r{<pushStream id="percWindow"/>}
```

Comment:
What is appearing in front of this? Could you share some example data you've collected? Thanks!